File: 1528-shuffle-string/1528-shuffle-string.java
class Solution {
    public String restoreString(String s, int[] indices) {
        StringBuilder str =new StringBuilder(s);
        for(int i=0;i<s.length();i++){
            str.setCharAt(indices[i],s.charAt(i));
        }
        return String.valueOf(str);
    }
}

File: 1528-shuffle-string/1528-shuffle-string.c
char * restoreString(char * s, int* indices, int indicesSize){
char *ptr=(char*)malloc((indicesSize+1)*sizeof(char));
    for(int i=0;i<indicesSize;i++){
        ptr[indices[i]]=s[i];
    }
    ptr[indicesSize]='\0';
    return ptr;
}